bliss/predict.py

```diff
@@ -9,8 +9,10 @@
 from torch import Tensor
 
 from bliss.catalog import FullCatalog
-from bliss.surveys.decals import DarkEnergyCameraLegacySurvey as DECaLS
-from bliss.surveys.decals import DecalsFullCatalog
+from bliss.surveys.decals import (
+    DarkEnergyCameraLegacySurvey as DECaLS,  # pylint: disable=cyclic-import
+)
+from bliss.surveys.decals import DecalsFullCatalog  # pylint: disable=cyclic-import
```

Comment:
Any way to avoid a cyclic import? These can be quite confusing.

Reply:
Yes, believe these are some ways:
1. move/refactor to third file
2. import within function, outside top-level
3. duplicate the code

I think 1 is cleanest and should be done in a future PR if possible. 2 is the simplest and can be done for this PR

Reply:
I actually ended up implementing a mix of 2 and 3, and an "option 4" that is removing the DECaLS -> DES dependency so that now DES can be thought of to be entirely "encapsulated" by DECaLS.